index.html

```diff
@@ -278,6 +329,8 @@ <h2 itemprop="description">Save your 2FA secrets, then use this to scan them aga
       <div id="app_label">Issuer (label)</div>
     </div>
 
+    <br>
+    <p>Please note that the advanced settings are not supported by the Google Authenticator app yet. All advanced settings are ignored by this app, and default values are used.</p>
```

Comment:
I am wondering if it is better to put this text up next to the checkbox for advanced options? It could become visible when the user checks the checkbox, like the other new form elements.. But only if it looks good, of course.

Reply:
Fully agree, fixed with: da5dda871ad7960a591955d8c5706c8ab93c016c